pkg/api/v1alpha1/cluster.go

```diff
@@ -494,6 +495,16 @@ func validateControlPlaneCertSANs(cfg *Cluster) error {
 	return nil
 }
 
+func validateControlPlaneAPIServerExtraArgs(clusterConfig *Cluster) error {
+	allowedFlags := map[string]string{"service-account-issuer": "", "service-account-jwks-uri": ""}
+	for k := range clusterConfig.Spec.ControlPlaneConfiguration.APIServerExtraArgs {
+		if _, ok := allowedFlags[k]; !ok {
+			return fmt.Errorf("invalid ControlPlaneConfiguration.APIServerExtraArgs; only 'service-account-issuer' and 'service-account-jwks-uri' flags are accepted currently")
```

Comment:
Can we collect errors and use an aggregate so the user doesn't have to repeatedly run the CLI? 

https://github.com/chrisdoherty4/aws-eks-anywhere/blob/ad52e5bf1ac9ffff2b05c79844bb1534cc230c6e/pkg/errors/aggregate.go#L15

Reply:
Good suggestion. Will update that in a separate PR

Reply:
Moved this validation to preflight validations to allow skipping it through `--skip-validations` option